content/en/tracing/setup/dotnet-framework.md

```diff
@@ -197,7 +205,9 @@ To configure the Tracer using a JSON file, create `datadog.json` in the instrume
 ```json
 {
     "DD_TRACE_AGENT_URL": "http://localhost:8126",
-    "DD_SERVICE_NAME": "MyService",
+    "DD_ENV": "prod",
+    "DD_SERVICE": "MyService",
+    "DD_VERSION": "abc123",
     "DD_ADONET_ENABLED": "false"
```

Comment:
```suggestion
    "DD_ADONET_ENABLED": "true"
```

Reply:
@lucaspimentel can we remove the `DD_ADONET_ENABLED` settings? Maybe it was important earlier, but we should probably show minimal configuration to get up and runniing

Reply:
Yes, I was just commenting on the other similar changes and this one with your comment was the last one, heh. I'll still copy/paste for completeness:

Having `DD_ADONET_ENABLED` was just an example of how users can change the default values. All integration are already enabled by default. Feel free to remove the line entirely if it's confusing. Also, we realized recently that it's case-sensitive on Linux, so it should be `DD_AdoNet_ENABLED`.